go/libkb/checkers.go

```diff
@@ -79,6 +79,27 @@ var CheckKex2SecretPhrase = Checker{
 	Hint: "It looks like there was a typo in the secret phrase. Please try again.",
 }
 
+var CheckPaperKeyPhrase = Checker{
+	F: func(s string) bool {
+		p := NewPaperKeyPhrase(s)
+		version, err := p.Version()
+		if err != nil {
+			G.Log.Debug("error getting paper key version: %s", err)
+			return false
+		}
```

Comment:
Right now the only possible error here is that the key is empty string, I think. Maybe just check for that explicitly? (In addition to moving up the valid words check?)